pkg/data/helpers.go

```diff
@@ -37,3 +38,8 @@ func getFiltersFromStruct(i interface{}, keyName *string) (*expression.KeyCondit
 	}
 	return kb, cb
 }
+
+func putItem(input *dynamodb.PutItemInput, dataInterface *Account) error {
```

Comment:
Still not sure I get the value of this.  Should this exist for query, scan, get, etc.?

Reply:
Added query

Reply:
Added get, scan as well